### PR TITLE
Add small edits and fix typos

### PR DESCRIPTION
**Pull request by willingc**
@Lukasa I intended to fix the missing space in "thank you", but wound up going through the file and doing a light edit. Just let me know what changes you would like :sunny:

### CODE REVIEW

File: source/contributing.rst
Comment (LilyFirefly) [inline source/contributing.rst:4]: I'm not sure the extra comma helps, but if others think it does that's fine by me,

File: source/contributing.rst
Comment (LilyFirefly) [inline source/contributing.rst]: I prefer the old text in this case.

File: source/contributing.rst
Comment (LilyFirefly) [inline source/contributing.rst:178]: The extra "system" seems unnecessary to me, but again happy if others disagree.

File: source/contributing.rst
Comment (LilyFirefly) [inline source/contributing.rst:180]: Again I'm not sure the comma helps, but don't mind much.

File: source/contributing.rst
Comment (LilyFirefly) [inline source/contributing.rst:12]: :+1:

File: source/contributing.rst
Comment (LilyFirefly) [inline source/contributing.rst:25]: :+1:

File: source/contributing.rst
Comment (LilyFirefly) [inline source/contributing.rst:129]: :+1:

File: source/contributing.rst
Comment (LilyFirefly) [inline source/contributing.rst:191]: :+1:

File: source/contributing.rst
Comment (LilyFirefly) [inline source/contributing.rst:219]: :+1:

**Comment by willingc** [comment]
Thanks for the speedy review @Ian-Foote. As an FYI, I use a professional grammar checker for the documentation that I do on Project Jupyter. The changes made here were based on feedback from the tool. I'm happy to revert the edits above. I went back and forth on the 'umbrella' change myself, so I'll change that back to the original. The commas and extra 'system' were recommended changes by the tool.

**Comment by alexwlchan** [comment]
@Lukasa I’d merge this, but _waves hands_ something about the way GitHub handles write permissions on protected branches.

**Review by alexwlchan — APPROVED** [review]

File: source/contributing.rst
Comment (Lukasa) [inline source/contributing.rst:25]: This is fab. In the future we should make these internal links to the relevant sections, but that's not a reason to not merge this.

**Review by Lukasa — APPROVED** [review]
Thanks so much @willingc, this looks great! Also thanks to @Ian-Foote and @alexwlchan for review. I've fixed up the darn merge permissions too. Might need to write a tool to automatically manage this, as I seem to get it wrong a lot.

File: source/contributing.rst
Comment (sethmlarson) [reply source/contributing.rst:180]: I would argue a comma isn't correct here.

File: source/contributing.rst
Comment (sethmlarson) [reply source/contributing.rst:191]: I think the "feedback is judged to apply still" sentence should just be rewritten, it was confusing the first time I read it and it's still confusing to read.

**Review by sethmlarson — COMMENTED** [review]
Looks good, still have a few suggestions.

**Pull request merged**
